cypress/support/e2e.ts

```diff
@@ -34,6 +34,7 @@ beforeEach(() => {
                 'surveys-results-visualizations': true,
                 'auto-redirect': true,
                 notebooks: true,
+                experimentation: true,
```

Comment:
don't think any experiments flag are still in use, are they?

Reply:
true